### ISSUE TITLE
Not able to produce Jmeter HTML Report in Release Pipeline with this extension & failed with error

### ISSUE THREAD

**Issue by goyalg8** [lakshaykaushik/PublishHTMLReport#37]
I am using Jmeter for performance load testing in Azure Release pipelines, I am using this task (PublishHTMLReport) but somehow it always get failed at this task with below error: 

2021-03-26T01:26:37.5513930Z ##[error]Unable to process command '##vso[task.addattachment type=firstscriptname;name=content;]$(Build.ArtifactStagingDirectory)/HTMLReports/content/js/dashboard-commons.js' successfully. Please reference documentation (http://go.microsoft.com/fwlink/?LinkId=817296)
2021-03-26T01:26:37.5526204Z ##[error]Value cannot be null. (Parameter 'Cannot upload task attachment file, attachment file location is not specified or attachment file not exist on disk'.

I did echo on this directory $(Build.ArtifactStagingDirectory) and $(system.DefaultWorkignDirectory) Directory of D:\a\r1\a\HTMLReports <DIR> . <DIR> .. <DIR> content 9,670 index.html <DIR> sbadmin2-1.0.7 2,998 statistics.json 2 File(s) 12,668 bytes 4 Dir(s) 14,701,731,840 bytes free.

Index.html is getting generated under the directory but somehow this HTML report extension does not work at all. I have tried few different path values in input section such as:

htmlType: 'Jmeter'
    JmeterReportsPath: '$(Build.SourcesDirectory)/LoadReports'

nothing is working. 
Somebody know any alternate method of producing Jmeter HTML report as tab within Azure DevOps.
Many Thanks

**Comment by lakshaykaushik** [comment]
Can you let me know:
1. The task how you have used it.
2. Jmeter version you are using
3. ls -lrt output on LoadReports folder.

**Comment by goyalg8** [comment]
The following tasks in Release pipeline are as below: 
1. Jmeter task version: 5.2.1
2. Command Line Script: 
echo 'Jmeter............'
jmeter -n -t _JmeterTest-AG\JmeterWebApp.jmx -l _JmeterTest-AG\Summary.jtl -e -o HTMLReports

3. Publish HTML Report Task (your extension)
htmlType: Jmeter
JmeterReportsPath: $(System.DefaultWorkingDirectory)/HTMLReports/index.html

That is all I have.
I did not get you when you said -Irt output on LoadReports folder. I have tried many different path values, nothing is working
